src/components/BlogPage/Cards/BlogCard.tsx

```diff
@@ -67,12 +67,12 @@ const BlogCard = ({
           </Typography>
         </Box>
       </Box>
-      <Image
+      {/* <Image
         src={"/imgs/blog/testImage.jpg"}
         width={150}
         height={150}
         alt="Image for blog card"
-      ></Image>
+      ></Image> */}
```

Comment:
for future reference, if the HTML or React element has no children, its self-closing

Reply:
im aware, must of auto-completed and i didnt notice

Reply:
ah fair, ye hate it when vscode autocorrects stuff like that